auth/AuthSample/Program.cs

```diff
@@ -351,11 +351,14 @@ public object AuthExplicit(string projectId, string jsonPath)
         // method.
         public object AuthExplicitComputeEngine(string projectId)
         {
-            var credential = GoogleCredential.FromComputeCredential();
-            var channel = new Grpc.Core.Channel(
-                LanguageServiceClient.DefaultEndpoint.ToString(),
-                credential.ToChannelCredentials());
-            var client = LanguageServiceClient.Create(channel);
+            LanguageServiceClientBuilder builder = new LanguageServiceClientBuilder

```

Comment:
In this case, I'd keep it multi-line, but still just have a `client` variable, calling `Build()` at the end.